client/tools/settings.coffee

```diff
@@ -81,3 +85,44 @@ defineTool
   click: ->
     return unless (room = currentRoom.get())?
     room.gridSnap.set not room.gridSnap.get()
+
+defineTool
+  name: 'Transparency'
+  category: 'color'
+  icon: 'highlighter'
+  help: 'Change transparency of pen'
+  active: ->
+    allowTransparency.get()
+  click: ->
+    allowTransparency.set not allowTransparency.get()
+    updateOpacity 1.0, allowTransparency.curValue
+  init: ->
+    updateOpacity 1.0, false
+
+
+# These values are chosen for no particular reason.  I saw that
+# 12.5 was a number you liked for highlighting Perhaps .25 should be 12.5
```

Comment:
Where did you see 12.5? Ah, 0x20 in #139. That's not me, but may be useful guideline. I'll need to experiment.

Reply:
√

Reply:
I leave this to your wysdom